plugins/base/src/main/kotlin/translators/documentables/DefaultPageCreator.kt

```diff
@@ -526,28 +527,29 @@ open class DefaultPageCreator(
                     .mapValues { if (it.value.any { it is DClasslike }) it.value.filter { it !is DTypeAlias } else it.value }
                     .toSortedMap(compareBy(nullsLast(String.CASE_INSENSITIVE_ORDER)) { it })
```

Comment:
Another problem with sorting map entries this way: a sorted map can have only one entry per key, and key uniqueness is determined by the comparator. Here the comparator is ignoring case so it will consider two group names with differing case equal and thus one group will overwrite the other in the sorted map.
Consider using `entries.sortedBy`

Reply:
Changed it to `entries.sortedBy`, all tests are green, YOLO

Now `fun bAr()` and `fun BaR()` will be in different "groups", whereas before they were under the same one

Reply:
> whereas before they were under the same one

There were two groups before because `groupBy` used `it.name`, but one of them was dropped by placing them under the same key in the `SortedMap`

Reply:
Yep, by "group" I meant `DivergentGroup`, not the result of `groupBy` - in other words, one line in the table

Now there will be two different lines, so you'll have two entries for `bitstream` and `bitStream`

![2023-02-08_19-20-30](https://user-images.githubusercontent.com/22685910/217617742-8662e56e-6d6d-452e-b428-55427ce445f3.png)